group_vars/ssh.yml

```
@@ -0,0 +1,112 @@
+---
+
+# I need someone to complete this list bellow because Im probably missing most of them
+ssh_roles:
+  - name: admin
+    hosts: all
+    users:
+      - centos
+      - stats
+      - galaxy
+      - root  # can be omitted for now, centos can `sudo su`
+
+  - name: stats
+    hosts:
+      - sn09
+      - sn06
+    users:
+      - stats
+
+  - name: galaxy
+    hosts:
+      - sn09
+      - sn06
+    users:
+      - galaxy
+
+ssh_users:
+  - user: galaxy
+    key:
+      - "{{ galaxy_user_public_key }}"
+    roles:
+      - admin
+
+  - user: bgruening
+    key:
+      - https://github.com/bgruening.keys
+    roles:
+      - admin
+
+  - user: mira-miracolu
+    key:
+      - https://github.com/mira-miracolu.keys
```

```suggestion
  - user: mira-miracoli
    key:
      - https://github.com/mira-miracoli.keys
```
like the instant pasta ;)